examples/logical_interconnects.go

```diff
@@ -19,11 +19,11 @@ func newFalse() *bool {
 func main() {
 	var (
 		clientOV        *ov.OVClient
-		id              = "7a8b37a6-bde0-4d06-8d20-6e574c50f39f"
+		id              = "e6f42b51-ba5a-430e-8b68-a9d1de223293"
 		macAddress      = "94:57:A5:67:2C:BE"
-		internalVlan    = "504"
-		interconnectURI = "/rest/interconnects/b6b7325f-666f-474f-a8f7-2c32b3c9faab"
-		externalVlan    = "504"
+		internalVlan    = "1052"
+		interconnectURI = "rest/ethernet-networks/577d8021-67e1-4d3e-92a1-fa20f61c812a"
```

Comment:
Why did this uri changed from interconnects to ethernet-networks..? Any reason

Reply:
i was testing something, forgot to put it back to defaults. I have fixed it, let me know